app/src/main/java/com/myvetpath/myvetpath/SubDetailsActivity.java

```diff
@@ -14,13 +14,18 @@
 public class SubDetailsActivity extends AppCompatActivity {
 
     Intent create_sub_activity;
+    MyDBHandler myDBHandler;
+    Submission currentSub;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_sub_details);
-        int position = getIntent().getIntExtra("pos", 0);
-        String title = getResources().getStringArray(R.array.subsTitles)[position];
+        myDBHandler = new MyDBHandler(this);
+
+        int caseId = getIntent().getIntExtra("caseID", 1);
```

Comment:
From the Design Document "Case ID: An integer number created by using the last two digits of the year and then a six digit runningtotal. It’s primary use is to allow a pathologist to easily recognizing what submission they are working on.May be dropped in the future when the Use of this app expands."